src/main/kotlin/app/cash/licensee/dependencyGraph.kt

```diff
@@ -161,6 +170,17 @@ private fun loadDependencyCoordinates(
   }
 }
 
+private fun ResolvedComponentResult.isPlatform(): Boolean {
+  val singleVariant = variants.singleOrNull() ?: return false
+  // https://github.com/gradle/gradle/issues/8854
+  val stringAttribute = Attribute.of(CATEGORY_ATTRIBUTE.name, String::class.java)
+  val category = singleVariant.attributes.getAttribute(stringAttribute) ?: return false
```

Comment:
Not sure the null coalescing return is needed here, since the `when`'s `else` does the same. No harm, though.